tests/python/test_cfg_continue.py

```diff
@@ -14,8 +14,5 @@ def foo():
                 x[p] = 1
                 continue
 
-            if state[p] != 0:
```

Comment:
If you remove this line, there's a chance that the `continue` get removed due to redundancy. 

Btw, this test case should verify that the codes following `continue` did not get executed. Can you file a separate PR to make a fix for this test? Thanks!

```
    x = ti.field(dtype=int, shape=1)
    state = ti.field(dtype=int, shape=1)
    @ti.kernel
    def foo():
        for p in range(1):
            if state[p] == 0:
                x[p] = 1
                continue

            x[p] = 100
```